app/experimenter/experiments/api/v3/serializers.py

```diff
@@ -69,8 +80,13 @@ class Meta:
             "slug",
             "status",
             "type",
+            "reject_feedback",
         )
 
+    def get_reject_feedback(self, obj):
+        if obj.status == Experiment.STATUS_REJECTED:
+            return ExperimentRapidRejectChangeLogSerializer(obj.changes.latest()).data
```

Comment:
Oh I see, I suppose this works incidentally, I might just change this to be 

```py
old_status=Experiment.STATUS_REVIEW, new_status=Experiment.STATUS_REJECTED
```

just to be explicit